systemd/coreos-installer-reboot-loaddev.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
+ConditionVirtualization=zvm
+ConditionPathExists=/run/coreos-installer-loaddev
+
+Requires=coreos-installer.target
+After=coreos-installer.target
+After=coreos-installer-reboot.service
+OnFailure=emergency.target
+OnFailureJobMode=replace-irreversibly
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/vmcp ipl loaddev
```

Comment:
Does this tell the hypervisor to do a hard reset or does the guest still have an orderly shutdown?

Reply:
Hardreset. I guess it's not a problem on zVM